### PR TITLE
Minor Changes

### PR DESCRIPTION
**Pull request by DragonEggBedrockBreaking**
I have made some small changes to some parts of the GitHub repository:

- Removed 'forge' from contributing.md
- Semi-rewrote readme, for parity with sodium's recent change (see the branch for this PR on my fork for a visual representation of the new README)
- Some changes in `fabric.mod.json`

Unlike what many may expect, I did **not** set the homepage to modrinth/curseforge, and you can find my reasoning in JellySquid's comment on [this sodium PR.](https://github.com/CaffeineMC/sodium-fabric/pull/379)

I set the project homepage to https://jellysquid.me/projects/ so that it would be more related to the mods.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,54 +4,121 @@
...
+
+The latest releases of Phosphor are published to our [Modrinth](https://modrinth.com/mods/phosphor) and
+[GitHub release](https://github.com/CaffeineMC/phosphor-fabric/releases) pages. Releases are considered by our team to be
+**suitable for general use**, but they are not guaranteed to be free of bugs and other issues.
+
+Usually, releases will be made available on GitHub slightly sooner than other locations.
+
+#### Bleeding-edge builds (unstable)
+
+[![GitHub build status](https://img.shields.io/github/workflow/status/CaffeineMC/phosphor-fabric/Java CI with Gradle/1.16.x/dev)](https://github.com/CaffeineMC/phosphor-fabric/actions/workflows/gradle.yml)
```
Comment (DragonEggBedrockBreaking) [inline README.md]: Idk how to make this work, can someone else help please?

File: README.md [hunk 2]
```diff
@@ -4,54 +4,121 @@
...
+will generally reflect the latest snapshot of development.
+
+Bleeding edge builds will often include unfinished code that hasn't been extensively tested. That code may introduce
+incomplete features, bugs, crashes, and all other kinds of weird issues. You **should not use these bleeding edge builds**
+unless you know what you are doing and are comfortable with software debugging. If you report issues using these builds,
+we will expect that this is the case. Caveat emptor.
+
+### CurseForge
+
+[![CurseForge downloads](http://cf.way2muchnoise.eu/full_394468_downloads.svg)](https://www.curseforge.com/minecraft/mc-mods/phosphor)
```
Comment (DragonEggBedrockBreaking) [inline README.md]: Idk how to change the shield to Phosphor curseforge, so again, help wanted?

**Review by DragonEggBedrockBreaking — COMMENTED** [review]
Idk how shields work, help needed?

**Comment by DragonEggBedrockBreaking** [comment]
- fixed one shield issue
- renamed GitHub actions things, like was done in sodium

File: README.md (hunk 2 above)
Comment (DragonEggBedrockBreaking) [reply README.md]: I'm stupid

**Comment by DragonEggBedrockBreaking** [comment]
Changed links from jellysquid3 to CaffeineMC

**Comment by DragonEggBedrockBreaking** [comment]
fixes #23

**Comment by DragonEggBedrockBreaking** [comment]
I bumped the loom version to 0.6 and changed `modCompile` to `modImplementation` so that this builds on JDK 16 + Gradle 7.0-rc1 (minimum gradle version for JDK 16). I believe both of these changes are backwards compatible. This is likely the last change I will do.

**Comment by DragonEggBedrockBreaking** [comment]
I have removed the homepage link in fabric.mod.json due to its recent removal in the sodium repo.

**Comment by DragonEggBedrockBreaking** [comment]
Removed the build.gradle changes as I implemented them in my new Gradle 7 PR

File: README.md [hunk 3]
```diff
@@ -1,58 +1,127 @@
...
+before they are remapped into stable intermediary names. If you are working in a developer environment and would
+like to add the mod to your game, you should prefer to use the `modRuntime` or `modCompile` configurations provided by
+Loom instead of these outputs.
 
 ---
 
 ### License
 
-Phosphor is licensed under GNU LGPLv3, a free and open-source license. For more information, please see the [license file](https://github.com/jellysquid3/phosphor-fabric/blob/1.16.x/dev/LICENSE.txt).
+Phosphor is licensed under GNU LGPLv3, a free and open-source license. For more information, please see the [license file](https://github.com/CaffeineMC/phosphor-fabric/blob/1.16.x/dev/LICENSE.txt).
```
Comment (PhiProven) [inline README.md]: Can we use a relative link for this?

Comment (DragonEggBedrockBreaking) [reply README.md]: Sure, I will try

Comment (DragonEggBedrockBreaking) [reply README.md]: Done

**Comment by DragonEggBedrockBreaking** [comment]
Wait, wtf is wrong with the commit name? GitHub being stupid...

**Comment by DragonEggBedrockBreaking** [comment]
Wait, I saw that the other 3 mods have license.txt, so I will rename the file for parity

**Comment by DragonEggBedrockBreaking** [comment]
Done

File: README.md [hunk 4]
```diff
@@ -1,58 +1,127 @@
...
 
-You will need to install JDK 8 in order to build Phosphor. You can either install this through a package manager such as
-[Chocolatey](https://chocolatey.org/) on Windows or [SDKMAN!](https://sdkman.io/) on other platforms. If you'd prefer to
-not use a package manager, you can always grab the installers or packages directly from
-[AdoptOpenJDK](https://adoptopenjdk.net/).
+#### Building with Gradle
 
-On Windows, the Oracle JDK/JRE builds should be avoided where possible due to their poor quality. Always prefer using
-the open-source builds from AdoptOpenJDK when possible.
+Phosphor ses a typical Gradle project structure and can be built by simply running the default `build` task. After Gradle
```
Comment (mrmangohands) [inline README.md]: typo on sees

File: README.md [hunk 5]
```diff
@@ -1,58 +1,127 @@
...
+projects. The basic overview is provided here for those familiar.
 
-If you're hacking on the code or would like to compile a custom build of Phosphor from the latest sources, you'll want
-to start here.
+#### Requirements
 
-#### Prerequisites
+- JRE 8 or newer (for running Gradle)
+- JDK 8 (optional)
+  - If you neither have JDK 8 available on your shell's path or installed through a supported package manager (such as
```
Comment (mrmangohands) [inline README.md]: Phosphor doesn't actually require java 8 specifically like sodium

File: README.md (hunk 4 above)
Comment (DragonEggBedrockBreaking) [reply README.md]: fixed (it is actually supposed to say 'uses')

File: README.md (hunk 5 above)
Comment (DragonEggBedrockBreaking) [reply README.md]: edited

**Comment by mrmangohands** [comment]
Merged manually as 736c4966e57976fd7b15663dbc6b7c1128c9161d

**Pull request closed without merging**